packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -7,20 +7,42 @@ import { ObjectOptions, Static, TSchema, Type } from "@sinclair/typebox";
 import { RevisionTagSchema } from "../../core";
 import { ChangesetLocalIdSchema, EncodedChangeAtomId } from "../modular-schema";
 import { unionOptions } from "../../codec";
-import { DetachIdOverrideType } from "./types";
+
+export enum DetachIdOverrideType {
+	/**
+	 * The detach effect is the inverse of the prior attach characterized by the accompanying `CellId`'s revision and
+	 * local ID.
+	 *
+	 * An override is needed in such a case to ensure that rollbacks and undos return tree content to the appropriate
+	 * detached root. It is also needed to ensure that cell comparisons work properly for undos.
+	 */
+	Unattach = 0,
+	/**
+	 * The detach effect is reapplying a prior detach.
+	 *
+	 * The accompanying cell ID is used in two ways:
+	 * - It indicates the location of the cell (including adjacent cell information) so that rebasing over this detach
+	 * can contribute the correct lineage information to the rebased mark.
+	 * - It specifies the revision and local ID that should be used to characterize the cell in the output context of
+	 * detach.
+	 */
+	Redetach = 1,
+}
 
 const noAdditionalProps: ObjectOptions = { additionalProperties: false };
 
-const CellCount = Type.Number();
+const CellCount = Type.Number({ multipleOf: 1, minimum: 0 });
```

Comment:
Is this allowed to be 0? Maybe it should start at 1?

Reply:
It could indeed be constrained to start at 1.

Reply:
```suggestion
const CellCount = Type.Number({ multipleOf: 1, minimum: 1 });
```